mp/src/game/client/hl2mp/c_hl2mp_player.cpp

```diff
@@ -627,16 +627,16 @@ void C_HL2MP_Player::HandleSpeedChanges( void )
 {
 	int buttonsChanged = m_afButtonPressed | m_afButtonReleased;
 
-	if( buttonsChanged & IN_SPEED )
+	if( buttonsChanged & IN_SPEED || buttonsChanged & IN_FORWARD || buttonsChanged & IN_BACK || buttonsChanged & IN_MOVELEFT || buttonsChanged & IN_MOVERIGHT)
```

Comment:
Isn't this just: `buttonsChanged & (IN_SPEED | IN_FORWARD | IN_BACK | IN_MOVELEFT | IN_MOVERIGHT)`? Similar adjustments can be made for the other two lines.

Reply:
Sure

Reply:
I only know how to use this because i've seen it used elsewhere, If you can manipulate it like that and make it look that much preetier then go ahead and change it

Reply:
@AdamTadeusz I've filled in the suggestion changes, up to you on applying them and double check on them.